src/pykeen/models/unimodal/rgcn.py

```diff
@@ -24,9 +24,55 @@
 class RGCN(
     ERModel[torch.FloatTensor, RelationRepresentation, torch.FloatTensor],
 ):
-    """An implementation of R-GCN from [schlichtkrull2018]_.
+    r"""An implementation of R-GCN from [schlichtkrull2018]_.
 
-    This model uses graph convolutions with relation-specific weights.
+    The Relational Graph Convolutional Network (R-GCN) [schlichtkrull2018] builds on an adapted GCN that can be applied
+    on multi-relational data such as Knowledge Graphs.
+    More concretely, the RGCN model consists of two major parts: i.) a GCN-based entity-encoder that computes
+    latent representations for entities and ii.) an arbitrary interaction model that computes the plausibility of facts.
+    The GCN employed by the entity-encoder is adapted to include typed edges.
+    The forward pass of the GCN is defined by:
+
+     .. math::
+
+        \textbf{e}_{i}^{l+1} = \sigma \left( \sum_{r \in \mathcal{R}}\sum_{j\in \mathcal{N}_{i}^{r}}
+        \frac{1}{c_{i,r}} \textbf{W}_{r}^{l} \textbf{e}_{j}^{l} + \textbf{W}_{0}^{l} \textbf{e}_{i}^{l}\right),
+
+    where $\mathcal{N}_{i}^{r}$ is the set of neighbors of node $i$ that are connected to
+    $i$ by relation $r$, $c_{i,r}$ is a fixed normalization constant (but it can also be introduced as an additional
+    parameter), and $\textbf{W}_{r}^{l} \in \mathbb{R}^{d^{(l)} \times d^{(l)}}$ and
+    $\textbf{W}_{0}^{l} \in \mathbb{R}^{d^{(l)} \times d^{(l)}}$ are weight matrices of the `l`-th layer of the
+    RGCN.
+    The encoder aggregates for each node $e_i$ the latent representations of its neighbors and its
+    own latent representation $e_{i}^{l}$ into a new latent representation $e_{i}^{l+1}$.
+    In contrast to standard GCN, RGCN defines relation specific transformations
+    $\textbf{W}_{r}^{l}$ which depend on the type and direction of an edge.
+    The interaction model computes the plausibility score given the node representations $\textbf{e}_{i}^{L}$ that are
+    computed by the last layer $L$ of the RGCN, i.e., for a given triple $(h,r,t) \in \mathcal{K}$, the
+    corresponding node representations $h:=e_i^L$ and $t:=e_j^L$ are used:
+
+    .. math::
+        f(h,r,t) = \textbf{h} \textbf{R}_{r} \textbf{t} \enspace,
+
+    where $\textbf{R}_{r} \in \mathbb{R}^{d \times d}$ is a diagonal matrix and $f(h,r,t)$ is the
+    interaction model of DistMult (DistMult was employed in the original work, however, the general approach is not
+    restricted to DistMult).
+    To reduce the number of parameters required for the relation-specific transformation matrices and to avoid
+    over-fitting, two regularization approaches are presented:
+
+    The first approach (_basis decomposition_), represents the relation-specific transformation matrices as a
```

Comment:
use intersphinx to link to the classes that implement it here

Reply:
instead of repeating the formula, we could also just link to the respective class (and make sure that the details are mentioned there).